de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/metadata/MetadataDisplayStrings.properties

```diff
@@ -18,4 +18,6 @@ incomplete = Some fields are invalid. Do you wish to continue ?
 newUnit = New Unit
 importMd = Import to User Metadata
 importMdMsg = This will import your existing metadata to the CyVerse metadata database. This cannot be undone. For more information, please see <a href='https://pods.iplantcollaborative.org/wiki/x/EISdAQ' target='_blank'>Using Metadata with the Cyverse Data Store</a>. Continue?
-importMdTooltip = Imports your existing metadata to the CyVerse metadata database. This action cannot be undone
\ No newline at end of file
+importMdTooltip = Imports your existing metadata to the CyVerse metadata database. This action cannot be undone.
+metadataLink = https://pods.iplantcollaborative.org/wiki/x/pRS
```

Comment:
(Only because Kathleen helped me with a wiki link for one of my tickets) Is there a `wiki.cyverse.org` equivalent of this link so we can move away from `iplantcollaborative`?